src/commands/server/start.ts

```diff
@@ -226,13 +227,20 @@ export default class Start extends Command {
       task: () => new Listr(cheTasks.serverLogsTasks(flags, true))
     }], listrOptions)
 
+    const eventTasks = new Listr([{
+      title: 'Start following events',
+      task: () => new Listr(cheTasks.namespaceEventsTask(flags, this, true))
+    }], listrOptions)
+
     try {
+      this.log(`Eclipse Che logs will be available in '${ctx.directory}'`)
```

Comment:
I faced some inconsistent behavior and planned to register an issue for that, but it seems it should be easy to solve and can be done here.
So, I think it would be better to print such an output after a platform checks tasks will be done,
otherwise user will be notified that Che Logs will be available but actually there is not chance for this to happen since user did not specified platform and there will not be even try to deploy Che